clouddriver-security/src/main/groovy/com/netflix/spinnaker/clouddriver/security/resources/ServerGroupNameable.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google, Inc.
+ * Copyright 2018 Google, Inc.
```

Comment:
Generally we don't update this after file creation.

Reply:
It's a new file

Reply:
OK, the github diff is genuinely horrible and I can't tell what's going on.

Reply:
s/horrible/the best